app/assets/stylesheets/kudos/components/_kudo.scss

```diff
@@ -21,6 +21,32 @@
   top: 1em;
 }
 
+.kudo__update {
+  position: absolute;
+  right: 1em;
+  text-align: right;
+  top: 4.25rem;
+}
+
+.edit-button {
+  @extend .btn;
+  height: 25px;
+  padding: 0;
+  text-align: center;
+  width: 50px;
+}
+
+.save-button {
+  @extend .btn;
+  @extend .btn-primary;
+}
+
+.kudo__input {
+  border: 1px solid #aaa;
```

Comment:
imo all colors should be aliased and stored somewhere else (they're basically magic numbers)